Dockerfile

```diff
@@ -154,4 +210,17 @@ COPY moodle-shib.conf /etc/apache2/conf-enabled/
 COPY docker-entrypoint.d/* /docker-entrypoint.d/
 COPY 000-default.conf /etc/apache2/sites-available/
 
+# add modified files for Language Icon
+COPY themes/maker-header.mustache /var/www/html/theme/maker/templates/header.mustache
+COPY themes/boost-language_menu.mustache /var/www/html/theme/boost/templates/language_menu.mustache
+COPY themes/updated-icon-designs-4.1/mod/ /var/www/html/theme/maker/pix_plugins/mod
```

Comment:
It would be very useful to add some comments about how we got those files. Did you write them or sent by Kate? So for future updates, we can trace them back to the source. Same for the fonts and favicon above.

Reply:
These mustache codes were modified to meet Kate's requirements based on the UX/UI specifications.